.eslintrc

```diff
@@ -20,7 +20,7 @@
     "no-throw-literal": 2,
     "no-self-compare": 2,
     "no-void": 2,
-    "no-unused-vars": 2,
+    "no-unused-vars": 1,
```

Comment:
eslint was, for some reason, incorrectly flagging variables as unused, but trying it today.. it works! so back to `2`.